src/main/java/io/strimzi/kafka/bridge/SourceBridgeEndpoint.java

```diff
@@ -80,11 +81,13 @@ protected void handleClose() {
      */
     protected void send(KafkaProducerRecord<String, byte[]> krecord, Handler<AsyncResult<RecordMetadata>> handler) {
 
-        log.debug("Sending to Kafka on topic {} at partition {} and key {}", krecord.topic(), krecord.partition(), krecord.key());
         if (handler == null) {
             this.producerSettledMode.write(krecord);
         } else {
-            this.producerUnsettledMode.write(krecord, handler);
+            log.debug("Sending to topic " + krecord.topic() + " at partition {}", krecord.partition());
```

Comment:
let's move this log line before the `if` statement, because it's the same for both producers.